clouds/aws/roles/roles.py

```diff
@@ -19,6 +19,36 @@ def iam_client(region=DEFAULT_AWS_REGION):
     return boto3.client(service_name="iam", region_name=region)
 
 
+def get_roles(client=None):
+    """
+    Get all roles from IAM.
+    Args:
+        client: A boto3 client for IAM.
+
+    Returns:
+         List of roles.
+    """
+    LOGGER.info("Retrieving all roles from IAM.")
+
+    client = client or iam_client()
+
+    response = client.list_roles(MaxItems=1000)
+    roles = response["Roles"]
+    is_truncated = response["IsTruncated"]
+    marker = response.get("Marker", None)
+
+    if not is_truncated:
+        return roles
+
+    while is_truncated:
```

Comment:
```suggestion
    while is_truncated := response["IsTruncated"]:
```

Reply:
Doing this leaves `is_truncated` as an unused variable. Modified it to `while response ["isTruncated"]:`